### PR TITLE
Upgrade faster-whisper

### PR DESCRIPTION
**Pull request by rpurdel**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,22 +11,18 @@ _login:
 
 build-summaries : _login
 	docker buildx build \
-	--build-arg="BASE_IMAGE_BUILD=nvidia/cuda:12.3.0-devel-ubuntu20.04" \
-	--build-arg="BASE_IMAGE_RUN=nvidia/cuda:12.3.0-runtime-ubuntu20.04" \
+	--build-arg="BASE_IMAGE_BUILD=nvidia/cuda:12.2.2-cudnn8-devel-ubuntu22.04" \
```
Comment (saghul) [inline Makefile:14]: Shouldn't we drop these now that they match the defaults?

File: Makefile [hunk 2]
```diff
@@ -11,22 +11,18 @@ _login:
...
+	--build-arg="BASE_IMAGE_RUN=nvidia/cuda:12.2.2-cudnn8-runtime-ubuntu22.04" \
 	--progress plain \
 	--push \
 	--platform ${PLATFORMS} \
-	--cache-from type=local,src=${CACHE_DIR} \
```
Comment (saghul) [inline Makefile:19]: Why drop these?

File: docker/rootfs/etc/apt/sources.list.d/python-ppa.list [hunk 3]
```diff
@@ -1,2 +1,2 @@
-deb https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu focal main
-deb-src https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu focal main
+deb https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy main
+deb-src https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy main
```
Comment (saghul) [inline docker/rootfs/etc/apt/sources.list.d/python-ppa.list:2]: We don't need the src, now that I see it.

File: Makefile (hunk 2 above)
Comment (rpurdel) [reply Makefile:19]: not really working

Comment (saghul) [reply Makefile:19]: 👍

**Review by saghul — APPROVED** [review]
🚀  No code changes necessary when updating faster-whisper? Impressive!

**Pull request merged**